languages/en.ini

```diff
@@ -1061,6 +1061,7 @@ Review by = "Review by"
 Reviews = "Reviews"
 Save = "Save"
 Save Comment = "Save Comment"
+save_and_close_editing = "Save and close editing"
```

Comment:
Laittaisin "Save and close editor". Ja koska tuo on ihan validi stringi sellaisenaankin, niin käännösavain sen mukaisesti:
`Save and close editor = "Save and close editor"`